2024-08-06 16:49:47,993 INFO    StreamThr :2069530 [internal.py:wandb_internal():86] W&B internal server running at pid: 2069530, started at: 2024-08-06 16:49:47.993132
2024-08-06 16:49:47,994 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: status
2024-08-06 16:49:47,994 INFO    WriterThread:2069530 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/run-1npvlywr.wandb
2024-08-06 16:49:47,995 DEBUG   SenderThread:2069530 [sender.py:send():382] send: header
2024-08-06 16:49:47,995 DEBUG   SenderThread:2069530 [sender.py:send():382] send: run
2024-08-06 16:49:48,147 INFO    SenderThread:2069530 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files
2024-08-06 16:49:48,147 INFO    SenderThread:2069530 [sender.py:_start_run_threads():1136] run started: 1npvlywr with start time 1722934187.993132
2024-08-06 16:49:48,161 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 16:49:48,161 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: check_version
2024-08-06 16:49:48,307 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 16:49:48,317 DEBUG   HandlerThread:2069530 [system_info.py:__init__():32] System info init
2024-08-06 16:49:48,317 DEBUG   HandlerThread:2069530 [system_info.py:__init__():47] System info init done
2024-08-06 16:49:48,317 INFO    HandlerThread:2069530 [system_monitor.py:start():194] Starting system monitor
2024-08-06 16:49:48,317 INFO    SystemMonitor:2069530 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 16:49:48,317 INFO    HandlerThread:2069530 [system_monitor.py:probe():214] Collecting system info
2024-08-06 16:49:48,317 INFO    SystemMonitor:2069530 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 16:49:48,318 INFO    SystemMonitor:2069530 [interfaces.py:start():190] Started disk monitoring
2024-08-06 16:49:48,318 INFO    SystemMonitor:2069530 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 16:49:48,319 INFO    SystemMonitor:2069530 [interfaces.py:start():190] Started memory monitoring
2024-08-06 16:49:48,319 INFO    SystemMonitor:2069530 [interfaces.py:start():190] Started network monitoring
2024-08-06 16:49:48,326 DEBUG   HandlerThread:2069530 [system_info.py:probe():196] Probing system
2024-08-06 16:49:48,327 DEBUG   HandlerThread:2069530 [system_info.py:_probe_git():181] Probing git
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_info.py:_probe_git():189] Probing git done
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_info.py:probe():244] Probing system done
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T08:49:48.326202', 'startedAt': '2024-08-06T08:49:47.987671', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/qwen2-1.5b', '--use_lora', 'True', '--init_lora_weights', 'pissa_niter_4', '--target_modules', 'q_proj,o_proj,gate_proj,down_proj,up_proj', '--lora_rank', '32', '--lora_alpha', '32', '--lora_dropout', '0', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.03', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1853.7279642857145, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 3863.591, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1843.442, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3113.789, 'min': 800.0, 'max': 5500.0}, {'current': 3568.645, 'min': 800.0, 'max': 5500.0}, {'current': 3899.382, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 3699.399, 'min': 800.0, 'max': 5600.0}, {'current': 3741.463, 'min': 800.0, 'max': 5600.0}, {'current': 4329.592, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 3105.211, 'min': 800.0, 'max': 5500.0}, {'current': 3494.379, 'min': 800.0, 'max': 5500.0}, {'current': 3269.944, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1411.319, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1647.183, 'min': 800.0, 'max': 4300.0}, {'current': 3580.87, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1100.017, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 440.3746681213379}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 16:49:48,329 INFO    HandlerThread:2069530 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 16:49:48,329 INFO    HandlerThread:2069530 [system_monitor.py:probe():227] Publishing system info
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 16:49:48,329 DEBUG   HandlerThread:2069530 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 16:49:49,148 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/conda-environment.yaml
2024-08-06 16:49:49,148 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/requirements.txt
2024-08-06 16:49:50,054 DEBUG   HandlerThread:2069530 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 16:49:50,054 DEBUG   HandlerThread:2069530 [system_info.py:_save_code():90] Saving code
2024-08-06 16:49:50,055 DEBUG   HandlerThread:2069530 [system_info.py:_save_code():111] Saving code done
2024-08-06 16:49:50,055 DEBUG   HandlerThread:2069530 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 16:49:50,058 ERROR   HandlerThread:2069530 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 16:49:50,058 DEBUG   HandlerThread:2069530 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 16:49:50,058 INFO    HandlerThread:2069530 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 16:49:50,059 DEBUG   SenderThread:2069530 [sender.py:send():382] send: files
2024-08-06 16:49:50,059 INFO    SenderThread:2069530 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 16:49:50,060 INFO    SenderThread:2069530 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 16:49:50,062 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 16:49:50,062 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: stop_status
2024-08-06 16:49:50,064 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 16:49:50,147 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/conda-environment.yaml
2024-08-06 16:49:50,147 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/wandb-metadata.json
2024-08-06 16:49:50,147 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/code/llama/gmm.py
2024-08-06 16:49:50,147 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/code
2024-08-06 16:49:50,148 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/code/llama
2024-08-06 16:49:50,285 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528
2024-08-06 16:49:50,286 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5486', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:50,286 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E917847E1E5A9D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:50,311 DEBUG   SenderThread:2069530 [sender.py:send():382] send: telemetry
2024-08-06 16:49:50,311 DEBUG   SenderThread:2069530 [sender.py:send():382] send: config
2024-08-06 16:49:50,312 DEBUG   SenderThread:2069530 [sender.py:send():382] send: metric
2024-08-06 16:49:50,312 DEBUG   SenderThread:2069530 [sender.py:send():382] send: telemetry
2024-08-06 16:49:50,312 DEBUG   SenderThread:2069530 [sender.py:send():382] send: metric
2024-08-06 16:49:50,312 WARNING SenderThread:2069530 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 16:49:50,312 DEBUG   SenderThread:2069530 [sender.py:send():382] send: telemetry
2024-08-06 16:49:50,313 DEBUG   SenderThread:2069530 [sender.py:send():382] send: telemetry
2024-08-06 16:49:50,313 DEBUG   SenderThread:2069530 [sender.py:send():382] send: config
2024-08-06 16:49:50,315 DEBUG   SenderThread:2069530 [sender.py:send():382] send: exit
2024-08-06 16:49:50,315 INFO    SenderThread:2069530 [sender.py:send_exit():589] handling exit code: 1
2024-08-06 16:49:50,315 INFO    SenderThread:2069530 [sender.py:send_exit():591] handling runtime: 1
2024-08-06 16:49:50,317 INFO    SenderThread:2069530 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 16:49:50,318 INFO    SenderThread:2069530 [sender.py:send_exit():597] send defer
2024-08-06 16:49:50,318 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,318 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 16:49:50,319 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,319 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 16:49:50,319 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 1
2024-08-06 16:49:50,319 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,319 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 16:49:50,319 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,319 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 16:49:50,320 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 2
2024-08-06 16:49:50,320 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,320 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 16:49:50,320 INFO    HandlerThread:2069530 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 16:49:50,322 DEBUG   SystemMonitor:2069530 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 16:49:50,322 DEBUG   SystemMonitor:2069530 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 16:49:50,322 DEBUG   SystemMonitor:2069530 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-06 16:49:50,324 INFO    HandlerThread:2069530 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 16:49:50,324 INFO    HandlerThread:2069530 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 16:49:50,397 INFO    HandlerThread:2069530 [interfaces.py:finish():202] Joined gpu monitor
2024-08-06 16:49:50,397 INFO    HandlerThread:2069530 [interfaces.py:finish():202] Joined memory monitor
2024-08-06 16:49:50,397 INFO    HandlerThread:2069530 [interfaces.py:finish():202] Joined network monitor
2024-08-06 16:49:50,397 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,397 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-06 16:49:50,397 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 3
2024-08-06 16:49:50,397 DEBUG   SenderThread:2069530 [sender.py:send():382] send: stats
2024-08-06 16:49:50,397 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,397 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-06 16:49:50,397 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,397 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-06 16:49:50,397 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 4
2024-08-06 16:49:50,398 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,398 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-06 16:49:50,398 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 5
2024-08-06 16:49:50,398 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,398 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-06 16:49:50,398 DEBUG   SenderThread:2069530 [sender.py:send():382] send: summary
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 16:49:50,398 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 6
2024-08-06 16:49:50,398 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,398 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-06 16:49:50,398 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,398 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-06 16:49:50,400 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 16:49:50,475 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2
2024-08-06 16:49:50,475 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '13927', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:50,475 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91784851C6F1D</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:50,562 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 7
2024-08-06 16:49:50,562 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:50,562 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-06 16:49:50,562 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:50,562 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-06 16:49:51,148 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/config.yaml
2024-08-06 16:49:51,149 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/wandb-summary.json
2024-08-06 16:49:51,149 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/output.log
2024-08-06 16:49:51,267 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-06 16:49:51,407 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528
2024-08-06 16:49:51,407 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5486', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:51,407 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91784C09101D9</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:51,745 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2
2024-08-06 16:49:51,745 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '13927', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:51,745 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E91784D3B49668</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:51,745 INFO    wandb-upload_1:2069530 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 16:49:52,328 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 8
2024-08-06 16:49:52,328 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: poll_exit
2024-08-06 16:49:52,328 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:52,328 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-06 16:49:52,329 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [job_builder.py:build():298] Attempting to build job artifact
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [job_builder.py:_get_source_type():439] no source found
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 9
2024-08-06 16:49:52,329 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:52,329 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-06 16:49:52,329 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 9
2024-08-06 16:49:52,329 INFO    SenderThread:2069530 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-06 16:49:53,149 INFO    Thread-12 :2069530 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/output.log
2024-08-06 16:49:53,151 INFO    SenderThread:2069530 [dir_watcher.py:finish():388] scan: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files
2024-08-06 16:49:53,151 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/wandb-summary.json wandb-summary.json
2024-08-06 16:49:53,151 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/requirements.txt requirements.txt
2024-08-06 16:49:53,156 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/output.log output.log
2024-08-06 16:49:53,156 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/config.yaml config.yaml
2024-08-06 16:49:53,156 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/wandb-metadata.json wandb-metadata.json
2024-08-06 16:49:53,156 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/conda-environment.yaml conda-environment.yaml
2024-08-06 16:49:53,167 INFO    SenderThread:2069530 [dir_watcher.py:finish():402] scan save: /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/code/llama/gmm.py code/llama/gmm.py
2024-08-06 16:49:53,167 INFO    SenderThread:2069530 [sender.py:transition_state():617] send defer: 10
2024-08-06 16:49:53,171 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: defer
2024-08-06 16:49:53,171 INFO    HandlerThread:2069530 [handler.py:handle_request_defer():172] handle defer: 10
2024-08-06 16:49:53,172 DEBUG   SenderThread:2069530 [sender.py:send_request():409] send_request: defer
2024-08-06 16:49:53,172 INFO    SenderThread:2069530 [sender.py:send_request_defer():613] handle sender defer: 10
2024-08-06 16:49:53,172 INFO    SenderThread:2069530 [file_pusher.py:finish():175] shutting down file pusher
2024-08-06 16:49:53,300 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904
2024-08-06 16:49:53,301 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:53,301 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/output.log</Resource><Region>wandb-local</Region><RequestId>17E9178530CFAE2B</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:53,316 INFO    wandb-upload_2:2069530 [upload_job.py:push():131] Uploaded file /home/rwkv/JL/GMM/llama/wandb/run-20240806_164947-1npvlywr/files/wandb-summary.json
2024-08-06 16:49:53,324 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74
2024-08-06 16:49:53,324 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:53,324 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E91785331427AF</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:53,435 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042
2024-08-06 16:49:53,436 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11992', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:53,436 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E9178539E417F7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:53,484 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a
2024-08-06 16:49:53,485 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:53,485 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E917853704326E</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:53,610 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528
2024-08-06 16:49:53,610 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5486', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:53,611 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9178543D8DAAE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:54,248 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2
2024-08-06 16:49:54,248 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '13927', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:54,248 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9178568EC9E07</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:54,248 INFO    wandb-upload_1:2069530 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 16:49:54,460 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904
2024-08-06 16:49:54,460 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:54,460 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/output.log</Resource><Region>wandb-local</Region><RequestId>17E9178575EED209</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:54,487 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74
2024-08-06 16:49:54,487 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:54,487 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E91785782E39AE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:54,789 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042
2024-08-06 16:49:54,790 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11992', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:54,790 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E91785864E3550</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:54,880 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a
2024-08-06 16:49:54,880 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:54,881 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E9178589A540FC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:56,269 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 16:49:56,930 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904
2024-08-06 16:49:56,931 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:56,931 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/output.log</Resource><Region>wandb-local</Region><RequestId>17E91786095E276A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:57,093 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042
2024-08-06 16:49:57,093 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11992', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:57,093 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E9178612B24704</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:57,255 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74
2024-08-06 16:49:57,255 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:57,255 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E91786093A255A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:57,558 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a
2024-08-06 16:49:57,558 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:57,558 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E91786221CDE9F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:58,173 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 16:49:58,496 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=1d90d2d44a9325fa257792a16cd4c128adc02018857b9195324c44bc17834528
2024-08-06 16:49:58,497 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5486', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:58,497 ERROR   wandb-upload_0:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E91786609DCFBE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:49:59,494 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084950Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=2c25569d23a7c4740b2ecd75616a440f63afdb3c596190bc4a487a6092205be2
2024-08-06 16:49:59,494 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '13927', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:49:59,495 ERROR   wandb-upload_1:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E917868A334830</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:50:01,271 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: keepalive
2024-08-06 16:50:01,390 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/output.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=88e3dc66d00b9c2fcb8ee96f041f1bbad620f3059a7ed239d1882a6308561904
2024-08-06 16:50:01,391 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '10561', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:50:01,391 ERROR   wandb-upload_4:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/output.log</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/output.log</Resource><Region>wandb-local</Region><RequestId>17E9178706859D96</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:50:01,637 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/requirements.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=4a568b6f91309d96d031d7be3345e09f7da36a4d44b0bd86d68ef3915cb53d74
2024-08-06 16:50:01,637 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:50:01,637 ERROR   wandb-upload_3:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/requirements.txt</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/requirements.txt</Resource><Region>wandb-local</Region><RequestId>17E917871CB1E6CE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:50:01,987 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/config.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=3bee16116f51d4527e82a3e1204ff7592b8c197d4c1e8c05b5d2d0c4f79ac042
2024-08-06 16:50:01,988 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '11992', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:50:01,988 ERROR   wandb-upload_5:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/config.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/config.yaml</Resource><Region>wandb-local</Region><RequestId>17E917872952E1CA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:50:02,626 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T084953Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=f9f8e276ccacc1e316ed9d0fe236e8a71069e9a8876fc7baf3aac9dc87de7e8a
2024-08-06 16:50:02,626 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '8681', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 16:50:02,626 ERROR   wandb-upload_6:2069530 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/1npvlywr/conda-environment.yaml</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/1npvlywr/conda-environment.yaml</Resource><Region>wandb-local</Region><RequestId>17E9178752D9411F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 16:50:03,174 DEBUG   HandlerThread:2069530 [handler.py:handle_request():146] handle_request: status_report
